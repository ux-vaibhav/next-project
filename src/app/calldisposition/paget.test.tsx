import React from 'react';
import { render } from '@testing-library/react';
import CallDispositionPage from './page'; // Adjust the import path accordingly

jest.mock('./components/call-disposition', () => ({
  CallDispositionForm: jest.fn(() => <div>Mocked CallDispositionForm</div>)
}));

describe('CallDispositionPage Component', () => {
  it('renders CallDispositionPage component', () => {
    const { getByText } = render(<CallDispositionPage />);

    // Replace the text content as needed based on your component's actual output
    expect(getByText('Mocked CallDispositionForm')).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<CallDispositionPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
