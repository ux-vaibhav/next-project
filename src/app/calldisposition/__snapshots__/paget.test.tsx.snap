// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallDispositionPage Component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      Mocked CallDispositionForm
    </div>
  </div>
</DocumentFragment>
`;
